scipy/special/special/wright_bessel.h

```diff
@@ -0,0 +1,737 @@
+        0.9566109552428079,  0.9728643851066921,  0.9853540840480059,  0.9940319694320907,   0.998866404420071};
+    /* weights for legendre polynomial of order 50
+     * sympy.integrals.quadrature.import gauss_legendre(50, 16)[1] */
+    constexpr double wb_w_legendre[] = {
+        0.002908622553155141, 0.006759799195745401, 0.01059054838365097, 0.01438082276148557,  0.01811556071348939,
+        0.02178024317012479,  0.02536067357001239,  0.0288429935805352,  0.03221372822357802,  0.03545983561514615,
+        0.03856875661258768,  0.0415284630901477,   0.04432750433880328, 0.04695505130394843,  0.04940093844946632,
+        0.05165570306958114,  0.05371062188899625,  0.05555774480621252, 0.05718992564772838,  0.05860084981322245,
+        0.05978505870426546,  0.06073797084177022,  0.06145589959031666, 0.06193606742068324,  0.06217661665534726,
+        0.06217661665534726,  0.06193606742068324,  0.06145589959031666, 0.06073797084177022,  0.05978505870426546,
+        0.05860084981322245,  0.05718992564772838,  0.05555774480621252, 0.05371062188899625,  0.05165570306958114,
+        0.04940093844946632,  0.04695505130394843,  0.04432750433880328, 0.0415284630901477,   0.03856875661258768,
+        0.03545983561514615,  0.03221372822357802,  0.0288429935805352,  0.02536067357001239,  0.02178024317012479,
+        0.01811556071348939,  0.01438082276148557,  0.01059054838365097, 0.006759799195745401, 0.002908622553155141};
+    /* Fitted parameters for optimal choice of eps
+     * Call: python _precompute/wright_bessel.py 4 */
+    constexpr double wb_A[] = {0.41037, 0.30833, 6.9952, 18.382, -2.8566, 2.1122};
+
+    SPECFUN_HOST_DEVICE inline double wright_bessel_integral(double a, double b, double x) {
+        /* 5. Integral representation
+         *
+         * K(a, b, x, r) = exp(-r + x * r^(-a) * cos(pi*a)) * r^(-b)
+         * * sin(x * r^(-a) * sin(pi*a) + pi * b)
+         * P(eps, a, b, x, phi) =
+         *                      * exp(eps * cos(phi) + x * eps^(-a) * cos(a*phi))
+         *                      * cos(eps * sin(phi) - x * eps^(-a) * sin(a*phi)
+         *                        + (1-b)*phi)
+         *
+         * Phi(a, b, x) = 1/pi * int_eps^inf K(a, b, x, r) * dr
+         *              + eps^(1-b)/pi * int_0^pi P(eps, a, b, x, phi) * dphi
+         *
+         * for any eps > 0.
+         *
+         * Note that P has a misprint in Luchko (2008).
+         * This integral representation introduced the free parameter eps (from the
+         * radius of complex contour integration). We try to choose eps such that
+         * the integrand behaves smoothly.
+         *
+         * As K has a leading exp(-r), we factor this out and apply Gauss-Laguerre
+         * quadrature rule:
+         *
+         * int_0^inf K(a, b, x, r+eps) dr = exp(-eps) int_0^inf exp(-r) Kmod(.., r) dr
+         *
+         * Note the shift r -> r+eps to have integation from 0 to infinity.
```

Comment:
```suggestion
         * Note the shift r -> r+eps to have integration from 0 to infinity.
```
tiny typo in the comment line